frontend/src/components/Form.test.js

```diff
@@ -45,3 +46,68 @@ describe('Form', () => {
     );
   });
 });
+
+describe('Form Validation', () => {
+  it('should return true when all fields are provided and time is compatible',
+      () => {
+        const role = 'Intern';
+        const productArea = 'Platforms and Ecosystems';
+        // Wed Jul 15 2020 20:56:45
+        const endTimeAvailableMilliseconds = 1594846605591;
+        const duration = 15;
+        // Wed Jul 15 2020 08:30:00
+        const currentTimeMilliseconds = 1594801800000;
+        expect(validateFormInputs(role, productArea, duration,
```

Comment:
```suggestion
        const currentTimeMilliseconds = 1594801800000;
        
        expect(validateFormInputs(role, productArea, duration,
```

newline between these two separate sections of the test. Here and below

Reply:
Made this change, and I have a follow up question to potentially clean it up more: Based on the arrange-act-assert test organization method, would this be cleaner if I made the method call and saved the value to a variable in a separate line above the assertion step?

Reply:
Something like:

// Arrange step with all values
`const role = 'Intern' `
...

// Act
`const inputValidation = validateFormInputs(role, productArea, duration, endTimeAvailableMilliseconds, currentTimeMilliseconds)`

// Assert 
`expect(inputValidation).toBe(true);`

Reply:
It couldn't hurt but it's not required if the expected value can be calculated concisely.

```
it('should add numbers', () => {
  expect(adder.add(2,2)).toBe(4)
})
```

is a perfectly valid test.

Reply:
Got it, thanks for the clarification!